Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1172,15 +1174,11 @@ This section elaborates on the various ``kubeProxyReplacement`` options:
 
     helm install cilium |CHART_RELEASE| \\
         --namespace kube-system \\
-        --set kubeProxyReplacement=partial \\
+        --set kubeProxyReplacement=false \\
         --set nodePort.enabled=true \\
         --set externalIPs.enabled=true
 
-- ``kubeProxyReplacement=disabled``: This option disables any Kubernetes service
-  handling by fully relying on kube-proxy instead, except for ClusterIP services
-  accessed from pods (pre-v1.6 behavior).
-
-In Cilium's Helm chart, the default mode is ``kubeProxyReplacement=disabled`` for
+In Cilium's Helm chart, the default mode is ``kubeProxyReplacement=false`` for
 new deployments.
```

Comment:
nit ... new deployments in order to support older kernels than v4.19.

Reply:
Ditto.